spec/models/ability_spec.rb

```diff
@@ -23,6 +23,10 @@
     it { should_not be_able_to(:update, UserMessageStatus.new) }
     it { should_not be_able_to(:create, Document.new) }
     it { should_not be_able_to(:update_settings, User.new) }
+
+    context 'with user management' do
+      it { is_expected.not_to be_able_to(:index, User) }
```

Comment:
Could we use `it { should_not be_able_to(:index, User) }` here, for consistency with other tests? (Same goes for the other new tests you've added.)

Reply:
Thanks. We are going the other way so I should probably have changed all the instances of `should` and `should_not` to `is_expected.to` and `is_expected.not_to` while I'm here.